### PR TITLE
Lab custom experiment

### LINKED ISSUE #73: Run lab on entire reformulation process

### PR DESCRIPTION
**Pull request by matblicher**
Closes #73

### CODE REVIEW

File: runP7.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+reportfolder = args.report if args.report else os.path.join(os.path.dirname(os.path.abspath(__file__)),"build/LabReport")
+
+projectfile = args.P7 if args.P7 else os.path.join(os.path.dirname(os.path.abspath(__file__)),"build/P7")
+downwardfilepath = args.downward
+validatorfilepath = args.validate
+
+search = args.search
+evaluator = args.evaluator
+
+benchmarksfolder = args.benchmarks if args.benchmarks else os.path.join(os.path.dirname(os.path.abspath(__file__)),"Data/Classical tracks/Gripper/")
```
Comment (kris701) [inline runP7.py:78]: Should this be here? Does it not lock it into **only** being able to use the Gripper problem/domain?
Was it intended to be a step back? Or maybe just not even on if a `--problem` or `--domain` argument was given?

**Review by kris701 — CHANGES_REQUESTED** [review]

File: runP7.py (hunk 1 above)
Comment (matblicher) [reply runP7.py:78]: If a folder for benchmarks is not given it defaults to /Data/Classical tracks/Gripper/ and the default for domain gripper_domain.pddl and problem gripper_problem.pddl.
Still need to make it run on multiple problems and make a custom parser for the result.

Comment (kris701) [reply runP7.py:78]: image
But you are always adding the benchmark folder to the path? So its not possible to select any other problem than gripper right now

Comment (matblicher) [reply runP7.py:78]: If you give the folder /Data/Classical tracks/Blockworld/
--benchmarks /path to/Data/Classical tracks/Blockworld/ --domain blocksworld.pddl --problem pb2.pddl

Comment (matblicher) [reply runP7.py:78]: Is it that you want it to take folder Data and run all domains and all problems inside?

Comment (kris701) [reply runP7.py:79]: oooooooooooooooooh i didnt see the `--benchmarks` tag! I just thought you gave the full path to problem/domain files.

Comment (matblicher) [reply runP7.py:79]: Oh okay, sorry I could have made it more clear which tags there are from the start.

**Review by kris701 — APPROVED** [review]

**Pull request merged**